### PR TITLE
reformatted the file with tabs

### PR DESCRIPTION
**Pull request by Chinay-Domitrix**

### CODE REVIEW

**Comment by edward-shen** [comment]
I appreciate the PR, but I'll going to decline these changes. In brief:

 - `rustfmt` by default uses four spaces and I'm inclined to respect the official formatter.
 - Markdown files are broken at 80 characters intentionally. 
 - Some whitespace is intentional, such as the additional whitespace [here](https://github.com/edward-shen/mangadex-home-rs/blob/master/src/main.rs#L78-L83) and between a header line and a body paragraph in markdown (which is more correct than not having the whitespace).

In conclusion, please respect the existing formatting. I will accept a separate PR with just the typo fixes, if you really want to contribute to this project.

Thanks!
